gazebo_ros_control:
   pid_gains:
      linear_arm_actuator_joint: {p: 200,  i: 0, d: 0, i_clamp: 100}
      shoulder_pan_joint:  {p: 350,  i: 0.0, d: 0, i_clamp: 10}
      shoulder_lift_joint: {p: 350,  i: 0.0, d: 0, i_clamp: 10}
      elbow_joint:         {p: 350,  i: 0.0, d: 0, i_clamp: 10}
      wrist_1_joint:       {p: 30,  i: 0.00, d: 0, i_clamp: 10}
      wrist_2_joint:       {p: 10,  i: 0.0, d: 0, i_clamp: 10}
      wrist_3_joint:       {p: 1.2,  i: 0.00, d: 0, i_clamp: 10}
      #wrist_3_joint:       {p: 1.2,  i: 0.0, d: 0, i_clamp: 1}
      #shoulder_pan_joint:  {p: 500,  i: 1, d: 1, i_clamp: 100}
      #shoulder_lift_joint: {p: 500,  i: 1, d: 1, i_clamp: 100}
      #elbow_joint:         {p: 500,  i: 1, d: 1, i_clamp: 100}
      #wrist_1_joint:       {p: 10,  i: 0.0, d: 0, i_clamp: 1}
      #wrist_2_joint:       {p: 10,  i: 0.0, d: 0, i_clamp: 1}
      #wrist_3_joint:       {p: 1,  i: 0.0, d: 0, i_clamp: 1}
      #no l
      #linear_arm_actuator_joint: {p: 150,  i: 1, d: 1, i_clamp: 100}
      #shoulder_pan_joint:  {p: 150,  i: 0, d: 0, i_clamp: 1}
      #shoulder_lift_joint: {p: 150,  i: 0, d: 0, i_clamp: 1}
      #elbow_joint:         {p: 150,  i: 0, d: 0, i_clamp: 1}
      #wrist_1_joint:       {p: 2,  i: 0.0, d: 0, i_clamp: 1}
      #wrist_2_joint:       {p: 2,  i: 0.0, d: 0, i_clamp: 1}
      #wrist_3_joint:       {p: 1,  i: 0.0, d: 0, i_clamp: 1}
      #shoulder_pan_joint:  {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #shoulder_lift_joint: {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #elbow_joint:         {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #wrist_1_joint:       {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #wrist_2_joint:       {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #wrist_3_joint:       {p: 1.2,  i: 0.0, d: 0.1, i_clamp: 1}
      #shoulder_pan_joint:  {p: 10000.0, i: 500.0, d: 200.0, i_clamp: 100.0}
      #shoulder_lift_joint: {p: 10000.0, i: 500.0, d: 200.0, i_clamp: 100.0}
      #elbow_joint:         {p: 10000.0, i: 500.0, d: 100.0, i_clamp: 100.0}
      #wrist_1_joint:       {p: 100.0, i: 0.0, d: 10.0, i_clamp: 0.0}
      #wrist_2_joint:       {p: 100.0, i: 0.0, d: 10.0, i_clamp: 0.0}
      #wrist_3_joint:       {p: 100.0, i: 0.0, d: 0.0, i_clamp: 0.0}
